app/main.py

```diff
@@ -1,13 +1,25 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
-def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+def format_single_linter_file(file_path: str,
+                              errors: list) -> dict:
+    return {
+        "errors":
+        [format_linter_error(error) for error in errors],
```

Comment:
```suggestion
        "errors": [format_linter_error(error) for error in errors],
```
I suppose it suitable by length